src/AD7792.c

```diff
@@ -163,10 +163,12 @@ float AD7792_getTemperature() {
   temp = AD7792_read(AD7792_COMM_ADDR(AD7792_REG_DATA), 2);
 
   /* linear interpolation from dummy values */
-  x1 = 30637.0;
-  y1 = 81.4;
-  x2 = 32440.0;
-  y2 = 100.9;
+  x1 = 23325.0;
+  y1 = 0;
+  x2 = 30637.0;
+  y2 = 81.4;
+  // x2 = 32440.0;
+  // y2 = 100.9;
```

Comment:
[please remove commented code](https://blog.kentcdodds.com/please-don-t-commit-commented-out-code-53d0b5b26d5f)

Reply:
Würde ich lassen - falls man mal den Sensor justieren möchte, kann man die Werte wieder gebrauchen.

Reply:
dead code does not belong in a commit (see the linked article above for a more detailed explanation)  
If you think we might need this in the future, we can keep it in the README.